Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/SimpleSearchController.java

```diff
@@ -172,33 +171,37 @@ private void refreshSearch(String comboValue) {
      * and as a short message the number of those processes.
      * @throws Exception
      */
-    private void processSearch() throws Exception {
+    private void processSearch() {
         SecurityService securityService = (SecurityService) SpringUtil.getBean("securityService");
-        if (securityService == null) {
-            throw new Exception("Security service unavailable");
-        }
         UserService userService = (UserService) SpringUtil.getBean("userService");
-        if (userService == null) {
-            throw new Exception("User service unavailable");
+        if (securityService == null || userService == null) {
+            Messagebox.show("Some internal services are not available",
+                "Error",
+                Messagebox.OK, Messagebox.ERROR);
```

Comment:
This failure of Manager services should be logged in addition to notifying the user.